<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Widget</class>
 <widget class="QWidget" name="Widget">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>747</width>
    <height>563</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Widget</string>
  </property>
  <widget class="QGroupBox" name="groupBox">
   <property name="geometry">
    <rect>
     <x>50</x>
     <y>130</y>
     <width>231</width>
     <height>241</height>
    </rect>
   </property>
   <property name="title">
    <string>字符串</string>
   </property>
   <widget class="QPushButton" name="btnAppend">
    <property name="geometry">
     <rect>
      <x>11</x>
      <y>31</y>
      <width>93</width>
      <height>28</height>
     </rect>
    </property>
    <property name="text">
     <string>Append</string>
    </property>
   </widget>
   <widget class="QPushButton" name="btnToUpper">
    <property name="geometry">
     <rect>
      <x>11</x>
      <y>66</y>
      <width>93</width>
      <height>28</height>
     </rect>
    </property>
    <property name="text">
     <string>toUpper</string>
    </property>
   </widget>
   <widget class="QPushButton" name="btnLeft">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>110</y>
      <width>93</width>
      <height>28</height>
     </rect>
    </property>
    <property name="text">
     <string>left</string>
    </property>
   </widget>
   <widget class="QPushButton" name="btnPrepend">
    <property name="geometry">
     <rect>
      <x>120</x>
      <y>30</y>
      <width>93</width>
      <height>28</height>
     </rect>
    </property>
    <property name="text">
     <string>Prepend</string>
    </property>
   </widget>
   <widget class="QPushButton" name="btnToLower">
    <property name="geometry">
     <rect>
      <x>120</x>
      <y>70</y>
      <width>93</width>
      <height>28</height>
     </rect>
    </property>
    <property name="text">
     <string>toLower</string>
    </property>
   </widget>
   <widget class="QPushButton" name="btnRight">
    <property name="geometry">
     <rect>
      <x>120</x>
      <y>110</y>
      <width>93</width>
      <height>28</height>
     </rect>
    </property>
    <property name="text">
     <string>right</string>
    </property>
   </widget>
   <widget class="QPushButton" name="btnSection">
    <property name="geometry">
     <rect>
      <x>120</x>
      <y>150</y>
      <width>93</width>
      <height>28</height>
     </rect>
    </property>
    <property name="text">
     <string>section</string>
    </property>
   </widget>
   <widget class="QPushButton" name="btnSimolified">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>200</y>
      <width>93</width>
      <height>28</height>
     </rect>
    </property>
    <property name="text">
     <string>simplified</string>
    </property>
   </widget>
   <widget class="QPushButton" name="btnTrimmed">
    <property name="geometry">
     <rect>
      <x>120</x>
      <y>200</y>
      <width>93</width>
      <height>28</height>
     </rect>
    </property>
    <property name="text">
     <string>trimmed</string>
    </property>
   </widget>
  </widget>
  <widget class="QGroupBox" name="groupBox_2">
   <property name="geometry">
    <rect>
     <x>310</x>
     <y>130</y>
     <width>121</width>
     <height>241</height>
    </rect>
   </property>
   <property name="title">
    <string>数字</string>
   </property>
   <widget class="QPushButton" name="btnCount">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>30</y>
      <width>93</width>
      <height>28</height>
     </rect>
    </property>
    <property name="text">
     <string>count</string>
    </property>
   </widget>
   <widget class="QPushButton" name="btnSize">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>80</y>
      <width>93</width>
      <height>28</height>
     </rect>
    </property>
    <property name="text">
     <string>size</string>
    </property>
   </widget>
   <widget class="QPushButton" name="btnIndexOf">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>130</y>
      <width>93</width>
      <height>28</height>
     </rect>
    </property>
    <property name="text">
     <string>indexOf</string>
    </property>
   </widget>
   <widget class="QPushButton" name="btnLastIndexOf">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>180</y>
      <width>93</width>
      <height>28</height>
     </rect>
    </property>
    <property name="text">
     <string>lastindexOf</string>
    </property>
   </widget>
  </widget>
  <widget class="QGroupBox" name="groupBox_3">
   <property name="geometry">
    <rect>
     <x>460</x>
     <y>130</y>
     <width>221</width>
     <height>241</height>
    </rect>
   </property>
   <property name="title">
    <string>逻辑判断</string>
   </property>
   <widget class="QPushButton" name="btnEndsWith">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>40</y>
      <width>93</width>
      <height>28</height>
     </rect>
    </property>
    <property name="text">
     <string>endsWith</string>
    </property>
   </widget>
   <widget class="QPushButton" name="btnStartsWith">
    <property name="geometry">
     <rect>
      <x>120</x>
      <y>40</y>
      <width>93</width>
      <height>28</height>
     </rect>
    </property>
    <property name="text">
     <string>startsWith</string>
    </property>
   </widget>
   <widget class="QPushButton" name="btnContains">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>100</y>
      <width>93</width>
      <height>28</height>
     </rect>
    </property>
    <property name="text">
     <string>contains</string>
    </property>
   </widget>
   <widget class="QPushButton" name="btnIsNull">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>170</y>
      <width>93</width>
      <height>28</height>
     </rect>
    </property>
    <property name="text">
     <string>isNull</string>
    </property>
   </widget>
   <widget class="QPushButton" name="btnIsEmpty">
    <property name="geometry">
     <rect>
      <x>120</x>
      <y>170</y>
      <width>93</width>
      <height>28</height>
     </rect>
    </property>
    <property name="text">
     <string>isEmpty</string>
    </property>
   </widget>
  </widget>
  <widget class="QWidget" name="layoutWidget">
   <property name="geometry">
    <rect>
     <x>50</x>
     <y>20</y>
     <width>631</width>
     <height>71</height>
    </rect>
   </property>
   <layout class="QFormLayout" name="formLayout">
    <item row="0" column="0">
     <widget class="QLabel" name="label">
      <property name="text">
       <string>str1</string>
      </property>
      <property name="buddy">
       <cstring>lineEditStr1</cstring>
      </property>
     </widget>
    </item>
    <item row="0" column="1">
     <widget class="QLineEdit" name="lineEditStr1"/>
    </item>
    <item row="1" column="0">
     <widget class="QLabel" name="label_2">
      <property name="text">
       <string>str2</string>
      </property>
      <property name="buddy">
       <cstring>lineEditStr2</cstring>
      </property>
     </widget>
    </item>
    <item row="1" column="1">
     <widget class="QLineEdit" name="lineEditStr2"/>
    </item>
   </layout>
  </widget>
  <widget class="QWidget" name="layoutWidget">
   <property name="geometry">
    <rect>
     <x>50</x>
     <y>410</y>
     <width>621</width>
     <height>23</height>
    </rect>
   </property>
   <layout class="QGridLayout" name="gridLayout">
    <item row="0" column="1">
     <widget class="QLineEdit" name="lineEditResult"/>
    </item>
    <item row="0" column="0">
     <widget class="QLabel" name="label_3">
      <property name="text">
       <string>结果</string>
      </property>
      <property name="buddy">
       <cstring>lineEditResult</cstring>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QWidget" name="layoutWidget">
   <property name="geometry">
    <rect>
     <x>50</x>
     <y>460</y>
     <width>621</width>
     <height>23</height>
    </rect>
   </property>
   <layout class="QHBoxLayout" name="horizontalLayout">
    <item>
     <widget class="QCheckBox" name="checkBoxContains">
      <property name="text">
       <string>contains()</string>
      </property>
     </widget>
    </item>
    <item>
     <spacer name="horizontalSpacer">
      <property name="orientation">
       <enum>Qt::Horizontal</enum>
      </property>
      <property name="sizeHint" stdset="0">
       <size>
        <width>40</width>
        <height>20</height>
       </size>
      </property>
     </spacer>
    </item>
    <item>
     <widget class="QLabel" name="label_4">
      <property name="text">
       <string>count()</string>
      </property>
      <property name="buddy">
       <cstring>spinBoxCount</cstring>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QSpinBox" name="spinBoxCount"/>
    </item>
   </layout>
  </widget>
  <zorder>layoutWidget</zorder>
  <zorder>layoutWidget</zorder>
  <zorder>layoutWidget</zorder>
  <zorder>groupBox</zorder>
  <zorder>groupBox_2</zorder>
  <zorder>groupBox_3</zorder>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections>
  <connection>
   <sender>lineEditStr1</sender>
   <signal>editingFinished()</signal>
   <receiver>btnContains</receiver>
   <slot>click()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>384</x>
     <y>51</y>
    </hint>
    <hint type="destinationlabel">
     <x>516</x>
     <y>243</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>lineEditStr2</sender>
   <signal>editingFinished()</signal>
   <receiver>btnContains</receiver>
   <slot>click()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>384</x>
     <y>79</y>
    </hint>
    <hint type="destinationlabel">
     <x>516</x>
     <y>243</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
